GRDB/QueryInterface/QueryInterfaceRequest.swift

```diff
@@ -39,9 +43,19 @@ extension QueryInterfaceRequest : FetchRequest {
     /// executed, and an eventual row adapter.
     ///
     /// - parameter db: A database connection.
+    /// - parameter hint: A hint about how the query should be prepared.
     /// - returns: A prepared statement and an eventual row adapter.
     /// :nodoc:
-    public func prepare(_ db: Database) throws -> (SelectStatement, RowAdapter?) {
+    public func prepare(_ db: Database, hint: FetchRequestHint?) throws -> (SelectStatement, RowAdapter?) {
+        let query: SQLSelectQuery
+
+        switch hint {
+        case .limitOne?:
+            query = self.query.limit(1)
```

Comment:
Here we break the following code:

```swift
let player = try Player.limit(1, offset: 10).fetchOne(db)
```

We have to honor the eventual offset, if present.

Reply:
Whoops! Should be fixed in the latest commits (and added a test to catch it).